docs/CloudStorages.md

```diff
@@ -30,24 +30,24 @@ Amazon Web Services S3 does not work with Hadoop 2.7. For this provider you'll n
 - If you are using a **public** bucket:
 ```bash
 $SPARK_HOME/bin/spark-shell \
-  --jars ./target/scala-2.12/qbeast-spark-assembly-0.2.0.jar \
-  --conf spark.sql.extensions=io.qbeast.spark.internal.QbeastSparkSessionExtension \
-  --conf spark.hadoop.fs.s3a.aws.credentials.provider=org.apache.hadoop.fs.s3a.AnonymousAWSCredentialsProvider \
-  --packages io.delta:delta-core_2.12:1.0.0,\
-    com.amazonaws:aws-java-sdk:1.12.20,\
-    org.apache.hadoop:hadoop-common:3.2.0,\
-    org.apache.hadoop:hadoop-client:3.2.0,\
-    org.apache.hadoop:hadoop-aws:3.2.0
+--conf spark.sql.extensions=io.qbeast.spark.internal.QbeastSparkSessionExtension \
+--conf spark.hadoop.fs.s3a.aws.credentials.provider=org.apache.hadoop.fs.s3a.AnonymousAWSCredentialsProvider \
+--packages io.qbeast:qbeast-spark_2.12:0.2.0,\
```

Comment:
if you add qbeast-spark as dependency, you also need to specify delta? I

Reply:
Yes, you do, as it is specified as a provided dependency:
https://github.com/Qbeast-io/qbeast-spark/blob/86c64812eb5a27801673e3d6b95a726c7005bb2a/build.sbt#L16

Reply:
I'm not sure using provided is the best solution here. Are we still using the assembly instead of the package? Once we publish in maven, we should migrate to use only the package, so all dependencies are downloaded directly. 
By the way, this command does not work for me, is the package published?

Reply:
No, is not published yet. And better, because then we can improve these :smile: 

Right now, we are using `assembly` behind the `package` command. I will try to shift and publish an alpha version on sonatype repository so we can test it out.